Code/Scripts/CMakeLists.txt

```diff
@@ -110,17 +112,29 @@ if(SV_USE_TKCXIMAGE)
 	endif()
 endif()
 
+if (SV_USE_SYSTEM_TCL)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+endif()
+
 if(NOT SV_USE_SYSTEM_TCL OR SV_INSTALL_SYSTEM_LIBS)
 	#Lib additions
-	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.5" developer_script_string)
-	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.5" developer_script_string)
-	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.5" install_script_string)
-	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.5" install_script_string)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.6" install_script_string)
+	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.6" install_script_string)
 endif()
 
 #--------
 # Python
 if(SV_USE_PYTHON)
+  if (SV_USE_SYSTEM_PYTHON)
+	   append_env_string_concat(PYTHONPATH "${PYTHON_SITE_PACKAGES}" developer_script_string)
+	if (SV_USE_SYSTEM_VTK)
```

Comment:
It didn't work without the addition of lines like these? VTK_DIR should be set irrespective of whether or not you are using system VTK. I think we have variable that are set for both system and not system that can be used, so that we don't need to differentiate between the two. At least that seems to work on mac and linux. Maybe there is something different on windows?